include/flatbuffers/util.h

```diff
@@ -210,6 +210,9 @@ inline void EnsureDirExists(const std::string &filepath) {
   auto parent = StripFileName(filepath);
   if (parent.length()) EnsureDirExists(parent);
   #ifdef _WIN32
+    #ifdef _MSC_VER
+      #pragma warning(suppress: 6031) // "return value ignored: <function> could return unexpected value"
```

Comment:
Ok, which call(s) cause this warning? If its few, can we fix it instead by pre-fixing `(void)` ?